src/util/timeTicks.js

```diff
@@ -0,0 +1,77 @@
+import { bisector, tickStep } from 'd3-array';
+import {
```

Comment:
Would it be possible to modify and use some of these unit tests?:  https://github.com/d3/d3-time/blob/main/test/ticks-test.js

Reply:
Added a unit test to check that the time ticks start on Monday.